integration-tests/grpc-proto3/disable-unbind-executions

```diff
@@ -0,0 +1 @@
+We need jar from this module to test supporting protos from external jars in grpc-external-proto-test
```

Comment:
What does this file do?

Reply:
> As I don't see it / find it in the changed files ...
> Or where was the indexing missing?

When I moved modules outside and actually package/install them, the problem with indexed classes is resolved. 

And the empty `beans.xml` file to run create jandex index of the module.

Reply:
> And the empty `beans.xml` file to run create jandex index of the module.

Ah, missed those.

@ozangunalp all good +1 !